### PR TITLE
Fix tenant name on the toast

### PR DESCRIPTION
**Pull request by bandinib-amzn**
*Issue #, if available:*

*Description of changes:*

- Tenant name on the toast resolved for global and private.

- Fix toast for View/Create Dashboard and View/Create Visualization.

**Before this fix:**
image
image

**After this fix:**
image
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/tenant-list/tenant-list.tsx [hunk 1]
```diff
@@ -130,24 +134,32 @@ export function TenantList(props: AppDependencies) {
     }
   };
 
-  const viewDashboard = async (tenantName: string) => {
+  const viewDashboard = async (tenantValue: string, action: string) => {
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/tenant-list/tenant-list.tsx:137]: It is a little bit weird the function name is still 'view' dashboard. Same comments apply to 'view' visualization

File: public/apps/configuration/panels/tenant-list/tenant-list.tsx [hunk 2]
```diff
@@ -130,24 +134,32 @@ export function TenantList(props: AppDependencies) {
...
     try {
-      await changeTenant(tenantName);
+      await changeTenant(tenantValue);
       window.location.href = getNavLinkById(props.coreStart, PageId.dashboardId);
     } catch (e) {
       console.log(e);
-      addToast(createUnknownErrorToast('viewDashboard', `view dashboard for ${tenantName} tenant`));
+      addToast(
+        createUnknownErrorToast(
+          'viewDashboard',
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/tenant-list/tenant-list.tsx:145]: Should id also be based on action?

**Comment by yuxitang-amzn** [comment]
There is typo `selet ${tenantName} tenant` in the code.

File: public/apps/configuration/panels/tenant-list/tenant-list.tsx (hunk 1 above)
Comment (bandinib-amzn) [reply public/apps/configuration/panels/tenant-list/tenant-list.tsx]: changed the function name to `viewOrCreateDashboard` and `viewOrCreateVisualization`

File: public/apps/configuration/panels/tenant-list/tenant-list.tsx (hunk 2 above)
Comment (bandinib-amzn) [reply public/apps/configuration/panels/tenant-list/tenant-list.tsx]: Right. changed id to be based on action.

**Comment by bandinib-amzn** [comment]
> There is typo `selet ${tenantName} tenant` in the code.

fixed.

**Pull request merged**